math/orpie/Portfile

```diff
@@ -18,17 +18,39 @@ long_description    Orpie is a fullscreen RPN calculator for the console. Its \
                     functionality, command completion, and a visible interactive \
                     stack.
 
-homepage            http://pessimization.com/software/orpie/
-master_sites        ${homepage}
+# not actually used for compiling -- OPAM is used instead
+homepage            https://github.com/pelzlpj/orpie/
+master_sites        ${homepage}/archive/refs/tags/
+distname            release-${version}
+worksrcdir          orpie-release-${version}
 
-checksums           rmd160 61ee7019ec5d141aade27d1c78770c2b11803085 \
-                    sha256 de557fc7f608c6cb1f44a965d3ae07fc6baf2b02a0d7994b89d6a0e0d87d3d6d
+checksums           release-1.6.1.tar.gz \
```

Comment:
Listing the name of the file in `checksums` is superfluous if there is only one distfile.